common/model/bundle.h

```diff
@@ -13,6 +13,14 @@
 #include "common/trinary/flex_trit.h"
 #include "utarray.h"
 
+#define TRYTE_HASH_LENGTH 81
+#define TRIT_HASH_LENGTH 243
```

Comment:
let's call this `HASH_LENGTH_TRIT` and `HASH_LENGTH_TRYTE`? also, `HASH_LENGTH` is already defined elsewhere. Are you sure the other values aren't as well?

Reply:
These values are actually spread and repeated across many files. What about a common definitions file ?

Reply:
I'm doing it in another PR, it's becoming huge.